modules/sdk-coin-sol/src/lib/ataInitializationBuilder.ts

```diff
@@ -51,18 +63,22 @@ export class AtaInitializationBuilder extends TransactionBuilder {
    * @param tokenName name of the token
    */
   mint(tokenName: string): this {
```

Comment:
should I add `deprecated` here now?
since in wp, it will be change to use `enableToken` as well